### DISCUSSION TITLE
LoRA errors after loading refiner model

### DISCUSSION THREAD

**Discussion by mozthefox** [Stability-AI/StableSwarmUI#265] [Q&A]
I turned on the CLI debug output and I've started to notice loads (100 maybe?) of LoRA error messages after Swarm loads the SDXL refiner model. Here's the first one - although they're all very similar:

`19:32:20.955 [Debug] ComfyUI-0 on port 7821 stderr: ERROR lora diffusion_model.input_blocks.4.1.proj_in.weight shape '[768, 768]' is invalid for input of size 409600`

I presume this is because the LoRA was trained using the main SDXL model and it isn't supported on the refiner model?

If not, or I'm doing something wrong, would someone kindly explain what I need to do to get rid of these error messages? (I can post the entire error output if needed).

Many thanks!
James

**Comment by mcmonkey4eva** [comment]
Debug output will always be spammed (that's why it's not visible normally), you can ignore it if it's not causing any actual practical issues for you.

**Reply by mozthefox** [reply]
Oh yeah, I understand that. I was just wondering why I was getting the errors. Everything seems to be running fine so I guess it's not causing an issue.

Thanks anyways! :)
